src/bgen/AttributeManager.cs

```diff
@@ -9,7 +9,7 @@
 
 public class AttributeManager {
 	public BindingTouch BindingTouch;
-	TypeManager TypeManager { get { return BindingTouch.TypeManager; } }
+	TypeCache TypeCache { get { return BindingTouch.TypeCache; } }
```

Comment:
minor: `TypeCache TypeCache => BindingTouch.TypeManager;`

Reply:
That'd be great for all the Manager objects, I'll get these in the next PR :)